python/python/tests/test_async_client.py

```diff
@@ -7102,6 +7102,119 @@ async def test_standalone_client_random_key(self, redis_client: GlideClient):
         # DB 0 should still have no keys, so random_key should still return None
         assert await redis_client.random_key() is None
 
+    @pytest.mark.parametrize("cluster_mode", [False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_lcs(self, redis_client: GlideClient):
+        min_version = "7.0.0"
+        if await check_if_server_version_lt(redis_client, min_version):
+            return pytest.mark.skip(reason=f"Redis version required >= {min_version}")
+        key1 = "testKey1"
+        value1 = "abcd"
+        key2 = "testKey2"
+        value2 = "axcd"
+        nonexistent_key = "nonexistent_key"
+        expected_subsequence = "acd"
```

Comment:
I don't get this -- why is acd the longest common subsequence? Shouldn't it be cd?

Reply:
This is `Longest common subsequence`, not to be confused with `Longest common substring`.